js/app.js

```
@@ -200,6 +200,31 @@ class CampaignManager {
         case "toggle-scene-tree":
           this.toggleSceneTreePanel();
           break;
+        case "navigate-to-scene":
+          const sceneId = target?.dataset.sceneId;
+          if (sceneId && this.sceneManager) {
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)